pkg/config/config.go

```diff
@@ -17,6 +17,11 @@ type Links struct {
 	Confirm string `default:"http://localhost/confirm/%v?token=%v"`
 }
 
+// OAuth2 is google config for oauth2 with google
+type OAuth2 struct {
+	State string `default:"random-state"`
```

Comment:
What is this state ?

Reply:
state is part of oAuth2 pattern, here to protect from XRSF attacks

Reply:
Ok! My bad :)